_config.yml

```diff
@@ -125,6 +125,13 @@ conference:
         Registration for the workshop is now open.
         Please visit the <b><a href="/registration">Registration</a></b> page for more
         information on how to register.
+    - title: Non-Author Travel Award Applications!
+      color: success
+      main_only: true
+      text: |
+        We're now accepting non-author travel awards applications.
```

Comment:
I would change this to "We are now accepting travel award applications for local non-author students"